lib/beaker-pe/pe-client-tools/install_helper.rb

```diff
@@ -37,7 +37,7 @@ def install_pe_client_tools_on(hosts, opts = {})
                 package_base << "-#{opts[:pe_client_tools_version]}"
                 package_name = package_base.dup
                 package_name << '-1' if opts[:pe_client_tools_version]
-                installer    = package_name + '-installer.pkg'
+                installer    = package_name + '*-installer.pkg'
```

Comment:
I think it might be `package_base` that needs the `*`. But not certain of that.

Reply:
It wasn't the package name that we need to glob, it was the name of the directory within `/Volumes`.